### PR TITLE
feat: Reload events when retry option is clicked

### PR DESCRIPTION
**Pull request by dreadpool2**
Fixes #275

### CODE REVIEW

File: app/src/main/java/org/fossasia/openevent/general/event/EventsFragment.kt [hunk 1]
```diff
@@ -95,6 +95,7 @@ class EventsFragment : Fragment() {
         showNoInternetScreen(isNetworkConnected())
 
         rootView.retry.setOnClickListener {
+            eventsViewModel.loadLocationEvents(eventsViewModel.savedLocation.toString())
```
Comment (simarsingh24) [inline app/src/main/java/org/fossasia/openevent/general/event/EventsFragment.kt]: check for network connect if present then call this

Comment (dreadpool2) [reply app/src/main/java/org/fossasia/openevent/general/event/EventsFragment.kt]: Retry button is visible only when network connection is not there

Comment (nikit19) [inline app/src/main/java/org/fossasia/openevent/general/event/EventsFragment.kt]: This will load events even when there is no internet connection right ?

Comment (dreadpool2) [reply app/src/main/java/org/fossasia/openevent/general/event/EventsFragment.kt]: Yup, it tries to load the events, but as there is an error loading events, Toast message pops up.

Comment (dreadpool2) [reply app/src/main/java/org/fossasia/openevent/general/event/EventsFragment.kt]: PR Updated. Please check

File: app/src/main/java/org/fossasia/openevent/general/event/EventsFragment.kt [hunk 2]
```diff
@@ -95,6 +95,8 @@ class EventsFragment : Fragment() {
         showNoInternetScreen(isNetworkConnected())
 
         rootView.retry.setOnClickListener {
+            if (isNetworkConnected())
+                eventsViewModel.loadLocationEvents(eventsViewModel.savedLocation.toString())
```
Comment (simarsingh24) [inline app/src/main/java/org/fossasia/openevent/general/event/EventsFragment.kt]: dont call `isNetworkConnected()`twice!

Comment (dreadpool2) [reply app/src/main/java/org/fossasia/openevent/general/event/EventsFragment.kt]: showNoInternetScreen function is used to show the screen. I am just calling this once actually :+1:

Comment (simarsingh24) [reply app/src/main/java/org/fossasia/openevent/general/event/EventsFragment.kt]: isNetworkConnected is being called in retryOnClickListener store it in some boolean and reuse

Comment (dreadpool2) [reply app/src/main/java/org/fossasia/openevent/general/event/EventsFragment.kt]: I will prevent this :+1:

Comment (dreadpool2) [reply app/src/main/java/org/fossasia/openevent/general/event/EventsFragment.kt]: Updated it

**Pull request merged**